Script/projectController.js

```diff
@@ -79,18 +79,26 @@ function getTicketWithProjectId(id)
 
 function createProjectPrompt()
 {
+    document.getElementById("projectModalHead").innerHTML = `
+            Project Creation Modal
```

Comment:
Use Speech Marks, also dont call it Modal, call it "Create Project"

Reply:
Done